src/inference/dev_api/openvino/runtime/threading/istreams_executor.hpp

```diff
@@ -139,7 +140,8 @@ class OPENVINO_RUNTIME_API IStreamsExecutor : virtual public ITaskExecutor {
                int threads = 0,
                PreferredCoreType threadPreferredCoreType = PreferredCoreType::ANY,
                std::vector<std::vector<int>> streamsInfoTable = {},
-               bool cpuReservation = false)
+               bool cpuReservation = false,
+               bool opt_denormals_for_tbb = true)
```

Comment:
I have to add this param in this constructor in order to fix CI issue.
`lto1: internal compiler error: in add_symbol_to_partition_1, at lto/lto-partition.c:153`